### PR TITLE
Articles can be fetched from API

### PR DESCRIPTION
**Pull request by stefankarlberg**
### [PT Story](https://www.pivotaltracker.com/story/show/166150759)
# Changes proposed in this PR:
* Adds request specs for articles API
* Adds unit tests for Article model
* Generates Article model, with following db columns: title, ingress, body and image
* Generates API Articles controller
* Nests articles resources into namespace v1 and namespace API in routes

# Things we learned:
* Refresher on binding.pry
* Refresher on API testing

### CODE REVIEW

**Review by leiter007 — COMMENTED** [review]
Great job guys, looks good to me - nothing to comment on!

**Comment by boamatule** [comment]
Pretty amazing. Follows all the recommendations from the pivotal tracker. Love the details in the indentation   too.

File: spec/requests/api/v1/articles_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+RSpec.describe Api::V1::ArticlesController, type: :request do
+  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+
+  describe 'GET /api/v1/article' do
+    before do
+      5.times { FactoryBot.create(:article)}
+    end
+
+    it 'returns an article' do
```
Comment (oliverochman) [inline spec/requests/api/v1/articles_spec.rb]: rewrite the description for this test

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: spec/requests/api/v1/articles_spec.rb (hunk 1 above)
Comment (stefankarlberg) [reply spec/requests/api/v1/articles_spec.rb]: Done!

**Review by oliverochman — APPROVED** [review]

**Pull request merged**